ArmPkg/Drivers/CpuDxe/PageTableMemoryAllocation.c

```diff
@@ -152,7 +153,7 @@ AllocatePageTableMemory (
   return Buffer;
 }
 
-PAGE_TABLE_MEM_ALLOC_PROTOCOL  mPageTableMemAllocProtocol = {
+STATIC PAGE_TABLE_MEM_ALLOC_PROTOCOL  mPageTableMemAllocProtocol = {

```

Comment:
```suggestion
static PAGE_TABLE_MEM_ALLOC_PROTOCOL  mPageTableMemAllocProtocol = {

```
https://edk2.groups.io/g/devel/topic/110977230#msg121076